src/ElectronNET.IntegrationTests/Tests/WebContentsTests.cs

```diff
@@ -75,9 +77,119 @@ public async Task Can_basic_print()
         [SkippableFact(Timeout = 20000)]
         public async Task GetPrintersAsync_check()
         {
-            Skip.If(Environment.GetEnvironmentVariable("GITHUB_TOKEN") != null, "Skipping printer test in CI environment.");
+            Skip.If(Environment.GetEnvironmentVariable("GITHUB_RUN_ID") != null, "Skipping printer test in CI environment.");
             var info = await fx.MainWindow.WebContents.GetPrintersAsync();
             info.Should().NotBeNull();
         }
```

Comment:
Windows server 2025 is the only platform where this takes longer, no need to completely disable.

Reply:
Sorry, I was on the wrong branch 🤣

Reply:
We can try to increase the timeout to 10s for GetPrintersAsync(). Most likely there's a Windiows service which is not started by default and takes some time.